.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,48 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build wheels and sdist on ${{ matrix.os }}
```

Comment:
could you maybe lengthen the name a bit to explain what sdist is? or put that in a comment?